packages/react-components/theme-designer/src/components/Sidebar/Sidebar.tsx

```diff
@@ -244,8 +92,16 @@ export const Sidebar: React.FC<SidebarProps> = props => {
           Edit
         </Tab>
       </TabList>
-      {tab === 'use' && <RenderUseTab />}
-      {tab === 'edit' && <RenderEditTab />}
+      {tab === 'use' && (
+        <UseTab
+          sidebarId={sidebarId}
```

Comment:
not a big deal, but this ID doesn't need to be shared. you can just useId within each component